pkg/arvo/gen/hood/fade.hoon

```diff
@@ -0,0 +1,17 @@
+::  Drum: destroy app
+::
+::::  /hoon/fade/hood/gen
+  ::
+/?    310
+::
+::::
+  ::
+:-  %say
+|=  $:  [now=@da eny=@uvJ bec=beak]
+        [arg=[@ $@(~ [@ ~])] ~]
+    ==
+:-  %drum-fade
+?>  ((sane %tas) -.arg)
```

Comment:
Why isn't `arg` just a `@tas` to begin with?

Reply:
each piece of `arg`, rather

Reply:
This is lifted exactly from `|start`.  Maybe they both should be updated to modern style.